content/applications/finance/expenses/reinvoice_expenses.rst

```diff
@@ -0,0 +1,144 @@
+Re-invoice expenses
+===================
+
+If expenses are tracked on customer projects, they can be automatically charged back to the
+customer. This is done by :ref:`creating an expense <expenses/reinvoice-create>`, referencing the
+sales order the expense is added to, and then :ref:`creating an expense report
+<expenses/reinvoice-report>`. Next, managers :ref:`approve the expense report
+<expenses/reinvoice-approve>`, before the accounting department :ref:`posts the journal entries
+<expenses/reinvoice-approve>`. Finally, once the expense report is posted to a journal, the expenses
+appears on the specified :abbr:`SO (Sales Order)`. The :abbr:`SO (Sales Order)` is then
+:ref:`invoiced <expenses/reinvoice>`, thus charging the customer for the expenses.
+
+.. important::
+   Approving expenses, posting expenses to accounting, and reinvoicing expenses on :abbr:`SOs (Sales
+   Orders)` is **only** possible for users with the *necessary* :doc:`access rights
+   <../../general/users/access_rights>`.
+
+.. seealso::
+   This document provides lower-level instructions for the creation, submission, approval, and
+   posting of expenses. For fully-detailed instructions for any of these steps, refer to the
+   following documentation:
+
+   - :doc:`Log expenses <../expenses/log_expenses>`
+   - :doc:`Expense reports <../expenses/expense_reports>`
+   - :doc:`Approving expenses <../expenses/approve_expenses>`
+   - :doc:`Posting expenses in accounting <../expenses/post_expenses>`
+
+Setup
+=====
+
+First, specify the invoicing policy for each expense category. Navigate to :menuselection:`Expenses
+app --> Configuration --> Expense Categories`. Click on an expense category to view the expense
+category form. Under the :guilabel:`INVOICING` section, click the radio button next to the desired
+selection for :guilabel:`Re-Invoice Expenses`:
+
+- :guilabel:`None`: The expense category cannot be re-invoiced.
+- :guilabel:`At cost`: The expense category invoices expenses at the cost set on the expense
+  category form.
+- :guilabel:`Sales price`: The expense category invoices at the price set on the expense form.
+
+.. _expenses/reinvoice-create:
+
+Create an expense
+=================
+
+First, when :doc:`creating a new expense <../expenses/log_expenses>`, the correct information needs
+to be entered to re-invoice the expense to a customer. Using the drop-down menu, select the
+:abbr:`SO (Sales Order)` to add the expense to in the :guilabel:`Customer to Reinvoice` field.
+
+Once a :abbr:`SO (Sales Order)` is attached to an expense report, and the report is saved, a
+:icon:`fa-money` :guilabel:`Sales Orders` smart button appears at the top-center of the screen.
+
+Next, select the :guilabel:`Analytic Distribution` the expense is posted to. Multiple accounts can
+be selected, if desired.
+
+To add another :guilabel:`Analytic Distribution`, click on the line to reveal the
+:guilabel:`Analytic` pop-over window. Click :guilabel:`Add a line`, then select the next
```

Comment:
```suggestion
:guilabel:`Analytic` pop-over window. Click :guilabel:`Add a line`, then select the desired
```
opting for the use of the word "desired" instead of "next" because I wouldn't want readers to assume they have to pick the very next account available, should that not be their intention.